package.json

```diff
@@ -17,8 +17,8 @@
     "start": "cd server && npm install && node start.js",
     "server": "PORT=8083 npm run dev --prefix server",
     "client": "npm start --prefix client",
-    "dev": "npm run server & npm run client",
-    "setup": "npm install --prefix server && npm install --prefix client",
+    "dev": "concurrently \"npm run server\" \"npm run client\"",
```

Comment:
Oh sweet. I wanted to do that. Good job.

Reply:
Right? The logs is much more pleasant to read now after this.